### PR TITLE
[Console] Update inherit and add licence

### PR DESCRIPTION
**Pull request by Simperfit**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no 
| BC breaks?    | no     
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets | none   
| License       | MIT
| Doc PR        | not needed 

see https://github.com/symfony/symfony/pull/32318/files#r299561556 it needs to be updated in the lowest branch.

### CODE REVIEW

File: src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php [hunk 1]
```diff
@@ -75,11 +75,7 @@ public function __construct($foreground = null, $background = null, array $optio
     }
 
     /**
-     * Sets style foreground color.
-     *
-     * @param string|null $color The color name
-     *
-     * @throws InvalidArgumentException When the color name isn't defined
```
Comment (ro0NL) [inline src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php]: should we move the `@throw`s to the contract instead?

File: src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php [hunk 2]
```diff
@@ -75,11 +75,7 @@ public function __construct($foreground = null, $background = null, array $optio
     }
 
     /**
-     * Sets style foreground color.
-     *
-     * @param string|null $color The color name
```
Comment (ro0NL) [inline src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php]: the `null` cases seem to be missing in the contract

File: src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php (hunk 1 above)
Comment (Simperfit) [reply src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php]: yes updated

File: src/Symfony/Component/Console/Formatter/OutputFormatterStyleInterface.php [hunk 3]
```diff
@@ -21,28 +23,36 @@ interface OutputFormatterStyleInterface
     /**
      * Sets style foreground color.
      *
-     * @param string $color The color name
+     * @param string|null $color The color name
+     *
+     * @throws InvalidArgumentException When the color name isn't defined
```
Comment (fabpot) [inline src/Symfony/Component/Console/Formatter/OutputFormatterStyleInterface.php]: Let's not add `@throws` annotations. We're not doing it everywhere and I don't want to go down this road.

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Console/Formatter/OutputFormatterStyleInterface.php (hunk 3 above)
Comment (Simperfit) [reply src/Symfony/Component/Console/Formatter/OutputFormatterStyleInterface.php]: removed

**Comment by Simperfit** [comment]
Status: Needs Review

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @Simperfit.

**Pull request merged**